DistroLauncher/DistroLauncher.cpp

```diff
@@ -37,6 +38,13 @@ HRESULT InstallDistribution(bool createUser, Oobe::Application<>& app)
         return hr;
     }
 
+    hr = AfterInstall(exitCode);
+#ifndef DNDEBUG
+    if (FAILED(hr)) {
+        return hr;
+    }
+#endif
+

```

Comment:
To minimize the diff we can instead of breaking the installation at this point, check for failures inside the `AfterInstall` function and log a debug warning.

Reply:
Makes sense to me. Done.